Wikipedia/Code/TalkPageViewController.swift

```diff
@@ -360,10 +364,17 @@ class TalkPageViewController: ViewController {
     }
     
     // MARK: - Overflow menu navigation
+
+    fileprivate func showUnexpectedErrorAlert() {
+        let alert = UIAlertController(title: TalkPageLocalizedStrings.unexpectedErrorAlertTitle, message: TalkPageLocalizedStrings.unexpectedErrorAlertSubtitle, preferredStyle: .alert)
+        let action = UIAlertAction(title: CommonStrings.okTitle, style: .default)
+        alert.addAction(action)
+        present(alert, animated: true)
+    }
     
     fileprivate func pushToDesktopWeb() {
         guard let url = viewModel.siteURL.wmf_URL(withPath: "/wiki/\(viewModel.pageTitle)", isMobile: false) else {
-            // TODO: Error banner
+            showUnexpectedErrorAlert()
```

Comment:
I'm adding the "Unexpected error" alert here, if it's something that only a connection error could cause, I'm happy to replace it with a "No connection" alert

Reply:
Thanks, I forgot about these. I don't think either error you've done so far quite fits here, since it doesn't really have anything to do with an internet connection or a server error. This is more a fallback in case for some reason it cannot create a URL to push the user to (like say, maybe there's a special character that causes the URL init to fail).

Can you call `showGenericError()` instead here? It's a basic error method in the superclass that displays a "Something went wrong, please try again later." banner.

Reply:
Good idea, done!